src/shogun2-core/src/main/java/de/terrestris/shogun2/service/PluginService.java

```diff
@@ -29,6 +34,11 @@
 	private String pluginNamespace;
 
 	/**
+	 * The application service which we e.g. need when plugins are deleted.
+	 */
+	private ApplicationService<Application, ApplicationDao<Application>> applicationService = null;
```

Comment:
As null is the default init value, this explicit initialization seems to be obsolete